### ISSUE TITLE
2.1.2 - React 16.14 / Typescript Error re: <Carousel> props

### ISSUE THREAD

**Issue by harleylang** [Learus/react-material-ui-carousel#85]
I am receiving the following error with the latest version bump to `2.1.2`

```
Failed to compile ...

No overload matches this call.
  Overload 1 of 2, '(props: CarouselProps, context?: any): ReactElement<any, any> | Component<CarouselProps, any, any> | null', gave the following error.
    Type '{ children: Element[]; }' is missing the following properties from type 'CarouselProps': navButtonsWrapperProps, navButtonsProps, NextIcon, PrevIcon, and 2 more.
  Overload 2 of 2, '(props: PropsWithChildren<CarouselProps>, context?: any): ReactElement<any, any> | Component<CarouselProps, any, any> | null', gave the following error.
    Type '{ children: Element[]; }' is missing the following properties from type 'CarouselProps': navButtonsWrapperProps, navButtonsProps, NextIcon, PrevIcon, and 2 more.  TS2769
```

In my component, I'm simply using an unstyled carousel with no custom buttons, eg.:

```
...
<Carousel>
<p> eg this does not work </p>
<Carousel>
...
```

**Comment by Learus** [comment]
Hello.

Firstly, this is not a problem with `v2.1.2`. It's a problem with the new version `2.2.0`.
It is now fixed, on version `2.2.1`. It was a problem with not having the new props optional. Please update to the new version.

Cheers.

**Comment by harleylang** [comment]
Thanks so much @Learus for providing a quick fix. I just tested `2.2.0` in my build and it works. Thanks again!
